File: read_pdf.py
import PyPDF2

def extract_text():
    with open('c:\\Users\\pc\\DEC MILESTONE TRACKER\\Performance Quality.pdf', 'rb') as file:
        reader = PyPDF2.PdfReader(file)
        text = ''
        for page in reader.pages:
            text += page.extract_text() + '\n'
        print(text)

extract_text()

File: read_pdf.mjs
import fs from 'fs';
import pdf from 'pdf-parse';

let dataBuffer = fs.readFileSync('c:\\Users\\pc\\DEC MILESTONE TRACKER\\Performance Quality.pdf');

pdf(dataBuffer).then(function (data) {
    console.log(data.text);
}).catch(function (error) {
    console.error(error);
});
